src/openforms/validations/registry.py

```diff
@@ -124,15 +69,15 @@ def validate(
                 ],
             )
 
-        if not getattr(validator.callable, "is_enabled", True):
+        if not getattr(validator, "is_enabled", True):
```

Comment:
this attribute should now always exist, otherwise the type checking would fail, right? And it's defined on the base class if I saw it right:

```suggestion
        if not validator.is_enabled:
```
makes grepping through the code a bit easier :)